### DISCUSSION TITLE
Allow multiple directories in DOCKGE_STACKS_DIR environment variable

### DISCUSSION THREAD

**Discussion by blacksteel1288** [louislam/dockge#696] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

For deployments of dockge where my docker-compose.yml files are scattered around in various directories, it would be great to be able to "point" to all of them to manage the compose files in place without moving them to a single directory.

### ✔️ Solution

For some apps that manage their compose.yml files in updates, this is necessary because moving them to another directory would break updates.

**Comment by SP3NGL3R** [comment]
could you use symlinks to the original compose folder inside the 'stacks' directory?
